data/pools/transactionPool.go

```diff
@@ -233,6 +233,20 @@ func (pool *TransactionPool) test(txgroup []transactions.SignedTxn) error {
 	// requires a flat MinTxnFee).
 	feePerByte = feePerByte * pool.feeThresholdMultiplier
 
+	// The feePerByte should be bumped to 1 to make the exponentially
+	// threshold growing valid.
+	if feePerByte == 0 && pool.numPendingWholeBlocks > 0 {
```

Comment:
This should be >1 since if we have just one whole block, we are not saturated yet..

Reply:
Make sense. Thanks!